packages/jest-util/src/create_process_object.js

```diff
@@ -9,7 +9,35 @@
 
 import deepCyclicCopy from './deep_cyclic_copy';
 
-const BLACKLIST = new Set(['mainModule', '_events']);
+const BLACKLIST = new Set(['env', 'mainModule', '_events']);
+
+function createProcessEnv() {
```

Comment:
I think it will be helpful to point to the `process.env` docs somewhere here (e.g. I wasn't aware of this, TIL): https://nodejs.org/api/process.html#process_process_env